pkg/universe.dagger.io/alpha/terraform/test/test.cue

```diff
@@ -44,6 +48,16 @@ dagger.#Plan & {
 
 			// TODO assert out.txt doesn't exist
 		}
+		importWorkflow: {
+			init: terraform.#Init & {
+				source: tfImportSource.output
+			}
+			importResource: terraform.#Import & {
+				source:  init.output
+				address: "random_uuid.test"
+				id:      "aabbccdd-eeff-0011-2233-445566778899"
```

Comment:
Where does this value come from? Is it a magic one?

Reply:
the address comes from the address of the terraform resource in the test file. the `id` is simple example uuid to allow importing the resource in terraform. It was the most simple resource that would need something to actually exit to import.

Reply:
Clear thank you!